Controls/Cafe/Cafe.h

```diff
@@ -8,6 +8,9 @@
  * your cafe challenge in Tests/Test_Cafe.c
  */
 
+// Function for checking error and printing if not NONE
```

Comment:
Use the JSDOC standard comment type:
/**
@brief what does it do
@param paramname what does it doe
...
*/